app/src/ui/notifications/pull-request-review-helpers.ts

```diff
@@ -0,0 +1,43 @@
+import { IAPIPullRequestReview } from '../../lib/api'
+import * as OcticonSymbol from '../octicons/octicons.generated'
+import { OcticonSymbolType } from '../octicons/octicons.generated'
+
+/** Returns the user-facing verb for a given review's state. */
+export function getVerbForPullRequestReview(review: IAPIPullRequestReview) {
+  switch (review.state) {
+    case 'APPROVED':
+      return 'approved'
+    case 'CHANGES_REQUESTED':
+      return 'requested changes on'
+    default:
+      return 'reviewed'
+  }
+}
+
+type ReviewStateIcon = {
+  symbol: OcticonSymbolType
+  className: string
+}
+
+/** Returns the icon info (symbol and class) for a given review's state. */
+export function getPullRequestReviewStateIcon<
+  T extends IAPIPullRequestReview['state']
+>(state: T): ReviewStateIcon {
```

Comment:
While working on https://github.com/desktop/desktop/pull/14215#discussion_r831019475 I ran into this, which doesn't make much sense, but I already changed it there 😂 

I could've done this:
```suggestion
export function getPullRequestReviewStateIcon(state: IAPIPullRequestReview['state']): ReviewStateIcon {
```
[I've learned some TypeScript-fu](https://github.com/type-challenges/type-challenges) since I wrote this, sorry 🤣

Reply:
Not a problem anymore, I just updated the PR with those changes.